src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -12,6 +12,10 @@
   display: none;
```

Comment:
An aside, but his rule gets inherited, no? https://github.com/Esri/calcite-app-components/blob/dfc9ba3aa7e28301e93f1d4825f8b1ad845b7cfd/src/assets/styles/_base.scss#L7-L9

Reply:
oh do I not need this in each component? @jcfranco

Reply:
I think we get that for free since we have the global styles set up. @asangma can you confirm?

Reply:
JC is correct

Reply:
rad